Gemfile

```diff
@@ -27,3 +27,4 @@ gem "sass-rails", ">= 6"
 gem "pg"
 gem "mysql2"
 gem "sqlite3"
+gem "cocoon"
```

Comment:
@excid3 I have used cocoon gem for dynamic nested form. I can see that we're not using jQuery so is it okay if I use [cocoon-vanilla-js](https://github.com/oddcamp/cocoon-vanilla-js), which is an equivalent JS written in vanilla-js?

Reply:
I think we can do it without cocoon. I did a screencast on that a bit ago: https://gorails.com/episodes/dynamic-nested-forms-with-stimulus-js

Reply:
Looks like screencast is not available for me but I'll try to implement it using stimulusjs. Thanks :)

Reply:
The source code is linked in the notes on that page: https://github.com/gorails-screencasts/dynamic-nested-forms-with-stimulusjs